powerhome/playbook
# [NUXE-1381] Tooltip position prop

____

#### Screens

<img width="898" alt="Screenshot 2023-03-28 at 2 26 03 PM" src="https://user-images.githubusercontent.com/8194056/228319880-c89ee286-1a13-436b-ad45-41671252c117.png">

#### Breaking Changes

No breaking changes, it adds the feature to change the default tooltip position from `absolute` to `fixed.

#### Runway Ticket URL

[[NUXE-1381]](https://nitro.powerhrg.com/runway/backlog_items/NUXE-1381)

#### How to test this

Open milano and check for the last example.
Run `yarn test-watch` on your local machine.

#### Checklist:
